### PR TITLE
feat(plugin): Fixed LastFMRichPresence albums reporting incorrectly

### PR DESCRIPTION
**Pull request by KontrollFreek**
For some reason LastFM decides to default to the last used album if it has no clue what it's supposed to be. Also did some grammar and capitalization stuff.

I had to make a new PR because I'm an idiot and deleted the main branch instead of just renaming it lmao

### CODE REVIEW

File: src/plugins/lastfm/index.tsx [hunk 1]
```diff
@@ -234,7 +239,7 @@ export default definePlugin({
             // why does the json api have xml structure
             return {
                 name: trackData.name || "Unknown",
-                album: trackData.album["#text"],
+                album: trackData.album.mbid === "" ? (trackData.name || "Unknown") : trackData.album["#text"],
```
Comment (Vendicated) [inline src/plugins/lastfm/index.tsx:242]: this doesn't seem right.. why use trackData.name here? isn't that the song name?

Comment (KontrollFreek) [reply src/plugins/lastfm/index.tsx:242]: This is just in case there's no album, usually indicating the song is a single and is therefore its own album

Comment (KontrollFreek) [reply src/plugins/lastfm/index.tsx:242]: i don't entirely remember but im fairly certain displaying nothing still displays something and i have no clue why honestly lmao

**Comment by Vendicated** [comment]
yeah this still doesn't look right, displaying the track (instead of just nothing) makes little sense

furthermore you have introduced so many unrelated changes :(

thanks a lot for your contribution but i'll be rejecting this

**Pull request closed without merging**